app/ide-desktop/lib/dashboard/src/authentication/src/authentication/components/resetPassword.tsx

```diff
@@ -0,0 +1,182 @@
+/** @file Container responsible for rendering and interactions in second half of forgot password
+ * flow. */
+import * as router from "react-router-dom";
+import toast from "react-hot-toast";
+
+import * as auth from "../providers/auth";
+import withRouter from "../../navigation";
+import * as hooks from "../../hooks";
+import * as utils from "../../utils";
+import * as app from "../../components/app";
+import * as icons from "../../components/svg";
+
+// =================
+// === Constants ===
+// =================
+
+const RESET_PASSWORD_QUERY_PARAMS = {
```

Comment:
consider merging with `RESET_PASSWORD_QUERY_PARAMS` and `LOGIN_QUERY_PARAMS`. see [this PR comment](https://github.com/enso-org/enso/pull/5864/files#r1133629495)

Reply:
(rejected for now, there's no real benefit)